### DISCUSSION TITLE
🚀 Poll: Supercharging Statamic with New Features!

### DISCUSSION THREAD

**Discussion by emran-alhaddad** [statamic/cms#9322] [Polls]
Hey Statamic Devs! 🛠️

I've got an idea brewing and need your genius minds to weigh in. We're talking about spicing up our Statamic experience with some cool new features:

Audit Log Extravaganza: Imagine tracking every twist and turn in your entries. Handy, right?
'Hey Admin, Heads Up!': A slick notification system that nudges admins when new drafts pop up. No more content getting lost in the shuffle!

@jackmcdade

**Comment by duncanmcclean** [comment]
Please leave feature requests on the dedicated `ideas` repository: http://github.com/statamic/ideas/issues

Other developers can 👍 an issue to vote if they think it’s worth implementing.

**Comment by ryanmitchell** [comment]
There is an audit log add on available on the marketplace: https://statamic.com/addons/mity-digital/logger
